pkg/registration/supply_chain_factory.go

```diff
@@ -177,8 +177,15 @@ func (f *SupplyChainFactory) createSupplyChain() ([]*proto.TemplateDTO, error) {
 	}
 	glog.V(4).Infof("Supply chain node: %+v", volumeSupplyChainNode)
 
+	businessAppSupplyChainNode, err := f.buildBusinessApplicationSupplyBuilder()
+	if err != nil {
+		return nil, err
+	}
+	glog.V(4).Infof("Supply chain node: %+v", businessAppSupplyChainNode)
+
 	supplyChainBuilder := supplychain.NewSupplyChainBuilder()
 	supplyChainBuilder.Top(serviceSupplyChainNode)
+	supplyChainBuilder.Entity(businessAppSupplyChainNode)
```

Comment:
businessapp should be the Top

Reply:
Thanks for the review @enlinxu. Updated, PTAL!